### PR TITLE
[SHELLEXT] localization of My Computer comments

### PR DESCRIPTION
**Pull request by KyleKatarn31**
## Purpose

- My Computer items's comments are not localized

image

JIRA issue: [CORE-17268](https://jira.reactos.org/browse/CORE-17268)

## Proposed changes

- Localization of My Computer items's comments

image

### CODE REVIEW

File: dll/shellext/ntobjshex/resource.h [hunk 1]
```diff
@@ -11,6 +11,8 @@
 
 #define IDS_NTOBJFLD_NAME        101
 #define IDS_REGISTRY_NAME        102
+#define IDS_REGISTRY_COMMENT     103
+#define IDS_NTOBJFLD_COMMENT     104
```
Comment (HBelusca) [inline dll/shellext/ntobjshex/resource.h:15]: Please sort the values (and translations above) in the natural order they were implemented (first, the NTOBJ stuff, then, the registry stuff).

Comment (KyleKatarn31) [reply dll/shellext/ntobjshex/resource.h:15]: OK

File: dll/shellext/ntobjshex/lang/cs-CZ.rc [hunk 2]
```diff
@@ -4,4 +4,6 @@ STRINGTABLE
 BEGIN
     IDS_NTOBJFLD_NAME "Jmenný prostor NT objektů"
     IDS_REGISTRY_NAME "Systémový registr"
+    IDS_REGISTRY_COMMENT "Allows browsing of the System Registry"
+    IDS_NTOBJFLD_COMMENT "Allows browsing of the NT Object Namespace"
```
Comment (HBelusca) [inline dll/shellext/ntobjshex/lang/cs-CZ.rc:8]:
```suggestion
    IDS_NTOBJFLD_COMMENT "Allows browsing of the NT Object Namespace"
    IDS_REGISTRY_COMMENT "Allows browsing of the System Registry"
```

File: dll/shellext/ntobjshex/lang/de-DE.rc [hunk 3]
```diff
@@ -4,4 +4,6 @@ STRINGTABLE
 BEGIN
     IDS_NTOBJFLD_NAME "NT-Objekt-Namespace"
     IDS_REGISTRY_NAME "Systemregistry"
+    IDS_REGISTRY_COMMENT "Allows browsing of the System Registry"
+    IDS_NTOBJFLD_COMMENT "Allows browsing of the NT Object Namespace"
```
Comment (HBelusca) [inline dll/shellext/ntobjshex/lang/de-DE.rc:8]:
```suggestion
    IDS_NTOBJFLD_COMMENT "Allows browsing of the NT Object Namespace"
    IDS_REGISTRY_COMMENT "Allows browsing of the System Registry"
```

File: dll/shellext/ntobjshex/lang/en-US.rc [hunk 4]
```diff
@@ -4,4 +4,6 @@ STRINGTABLE
 BEGIN
     IDS_NTOBJFLD_NAME "NT Object Namespace"
     IDS_REGISTRY_NAME "System Registry"
+    IDS_REGISTRY_COMMENT "Allows browsing of the System Registry"
+    IDS_NTOBJFLD_COMMENT "Allows browsing of the NT Object Namespace"
```
Comment (HBelusca) [inline dll/shellext/ntobjshex/lang/en-US.rc:8]:
```suggestion
    IDS_NTOBJFLD_COMMENT "Allows browsing of the NT Object Namespace"
    IDS_REGISTRY_COMMENT "Allows browsing of the System Registry"
```

File: dll/shellext/ntobjshex/lang/es-ES.rc [hunk 5]
```diff
@@ -4,4 +4,6 @@ STRINGTABLE
 BEGIN
     IDS_NTOBJFLD_NAME "Espacio de nombres NT"
     IDS_REGISTRY_NAME "Registro del sistema"
+    IDS_REGISTRY_COMMENT "Allows browsing of the System Registry"
+    IDS_NTOBJFLD_COMMENT "Allows browsing of the NT Object Namespace"
```
Comment (HBelusca) [inline dll/shellext/ntobjshex/lang/es-ES.rc:8]:
```suggestion
    IDS_NTOBJFLD_COMMENT "Allows browsing of the NT Object Namespace"
    IDS_REGISTRY_COMMENT "Allows browsing of the System Registry"
```

File: dll/shellext/ntobjshex/lang/fr-FR.rc [hunk 6]
```diff
@@ -4,4 +4,6 @@ STRINGTABLE
 BEGIN
     IDS_NTOBJFLD_NAME "Espace de noms des objets NT"
     IDS_REGISTRY_NAME "Registre système"
+    IDS_REGISTRY_COMMENT "Permet de naviguer dans le registre système"
+    IDS_NTOBJFLD_COMMENT "Permet de naviguer dans l'espace des objets NT"
```
Comment (HBelusca) [inline dll/shellext/ntobjshex/lang/fr-FR.rc:8]:
```suggestion
    IDS_NTOBJFLD_COMMENT "Permet de naviguer dans l'espace des objets NT"
    IDS_REGISTRY_COMMENT "Permet de naviguer dans le registre système"
```

File: dll/shellext/ntobjshex/lang/hi-IN.rc [hunk 7]
```diff
@@ -11,4 +11,6 @@ STRINGTABLE
 BEGIN
     IDS_NTOBJFLD_NAME "एनटी ऑब्जेक्ट नाम स्थान"
     IDS_REGISTRY_NAME "सिस्टम रजिस्ट्री"
+    IDS_REGISTRY_COMMENT "Allows browsing of the System Registry"
+    IDS_NTOBJFLD_COMMENT "Allows browsing of the NT Object Namespace"
```
Comment (HBelusca) [inline dll/shellext/ntobjshex/lang/hi-IN.rc:15]:
```suggestion
    IDS_NTOBJFLD_COMMENT "Allows browsing of the NT Object Namespace"
    IDS_REGISTRY_COMMENT "Allows browsing of the System Registry"
```

File: dll/shellext/ntobjshex/lang/it-IT.rc [hunk 8]
```diff
@@ -4,4 +4,6 @@ STRINGTABLE
 BEGIN
     IDS_NTOBJFLD_NAME "Oggetto Namespace NT"
     IDS_REGISTRY_NAME "Registro di Sistema"
+    IDS_REGISTRY_COMMENT "Allows browsing of the System Registry"
+    IDS_NTOBJFLD_COMMENT "Allows browsing of the NT Object Namespace"
```
Comment (HBelusca) [inline dll/shellext/ntobjshex/lang/it-IT.rc:8]:
```suggestion
    IDS_NTOBJFLD_COMMENT "Allows browsing of the NT Object Namespace"
    IDS_REGISTRY_COMMENT "Allows browsing of the System Registry"
```

File: dll/shellext/ntobjshex/lang/pl-PL.rc [hunk 9]
```diff
@@ -4,4 +4,6 @@ STRINGTABLE
 BEGIN
     IDS_NTOBJFLD_NAME "Przestrzeń nazw NT Object"
     IDS_REGISTRY_NAME "Rejestr systemu"
+    IDS_REGISTRY_COMMENT "Allows browsing of the System Registry"
+    IDS_NTOBJFLD_COMMENT "Allows browsing of the NT Object Namespace"
```
Comment (HBelusca) [inline dll/shellext/ntobjshex/lang/pl-PL.rc:8]:
```suggestion
    IDS_NTOBJFLD_COMMENT "Allows browsing of the NT Object Namespace"
    IDS_REGISTRY_COMMENT "Allows browsing of the System Registry"
```

File: dll/shellext/ntobjshex/lang/pt-PT.rc [hunk 10]
```diff
@@ -4,4 +4,6 @@ STRINGTABLE
 BEGIN
     IDS_NTOBJFLD_NAME "NT Espaço de nomes"
     IDS_REGISTRY_NAME "Registo do sistema"
+    IDS_REGISTRY_COMMENT "Allows browsing of the System Registry"
+    IDS_NTOBJFLD_COMMENT "Allows browsing of the NT Object Namespace"
```
Comment (HBelusca) [inline dll/shellext/ntobjshex/lang/pt-PT.rc:8]:
```suggestion
    IDS_NTOBJFLD_COMMENT "Allows browsing of the NT Object Namespace"
    IDS_REGISTRY_COMMENT "Allows browsing of the System Registry"
```

File: dll/shellext/ntobjshex/lang/ro-RO.rc [hunk 11]
```diff
@@ -6,4 +6,6 @@ STRINGTABLE
 BEGIN
     IDS_NTOBJFLD_NAME "Spațiul de nume Obiect NT"
     IDS_REGISTRY_NAME "Registrul de sistem"
+    IDS_REGISTRY_COMMENT "Allows browsing of the System Registry"
+    IDS_NTOBJFLD_COMMENT "Allows browsing of the NT Object Namespace"
```
Comment (HBelusca) [inline dll/shellext/ntobjshex/lang/ro-RO.rc:10]:
```suggestion
    IDS_NTOBJFLD_COMMENT "Allows browsing of the NT Object Namespace"
    IDS_REGISTRY_COMMENT "Allows browsing of the System Registry"
```

File: dll/shellext/ntobjshex/lang/ru-RU.rc [hunk 12]
```diff
@@ -6,4 +6,6 @@ STRINGTABLE
 BEGIN
     IDS_NTOBJFLD_NAME "Пространство имен объектов NT"
     IDS_REGISTRY_NAME "Системный реестр"
+    IDS_REGISTRY_COMMENT "Allows browsing of the System Registry"
+    IDS_NTOBJFLD_COMMENT "Allows browsing of the NT Object Namespace"
```
Comment (HBelusca) [inline dll/shellext/ntobjshex/lang/ru-RU.rc:10]:
```suggestion
    IDS_NTOBJFLD_COMMENT "Allows browsing of the NT Object Namespace"
    IDS_REGISTRY_COMMENT "Allows browsing of the System Registry"
```

File: dll/shellext/ntobjshex/lang/tr-TR.rc [hunk 13]
```diff
@@ -6,4 +6,6 @@ STRINGTABLE
 BEGIN
     IDS_NTOBJFLD_NAME "NT Nesne Ad Alanı"
     IDS_REGISTRY_NAME "Sistem Kayıt Kütüğü"
+    IDS_REGISTRY_COMMENT "Allows browsing of the System Registry"
+    IDS_NTOBJFLD_COMMENT "Allows browsing of the NT Object Namespace"
```
Comment (HBelusca) [inline dll/shellext/ntobjshex/lang/tr-TR.rc:10]:
```suggestion
    IDS_NTOBJFLD_COMMENT "Allows browsing of the NT Object Namespace"
    IDS_REGISTRY_COMMENT "Allows browsing of the System Registry"
```

File: dll/shellext/ntobjshex/lang/zh-CN.rc [hunk 14]
```diff
@@ -6,4 +6,6 @@ STRINGTABLE
 BEGIN
     IDS_NTOBJFLD_NAME "NT 对象名称空间"
     IDS_REGISTRY_NAME "系统注册表"
+    IDS_REGISTRY_COMMENT "Allows browsing of the System Registry"
+    IDS_NTOBJFLD_COMMENT "Allows browsing of the NT Object Namespace"
```
Comment (HBelusca) [inline dll/shellext/ntobjshex/lang/zh-CN.rc:10]:
```suggestion
    IDS_NTOBJFLD_COMMENT "Allows browsing of the NT Object Namespace"
    IDS_REGISTRY_COMMENT "Allows browsing of the System Registry"
```

File: dll/shellext/ntobjshex/lang/zh-TW.rc [hunk 15]
```diff
@@ -6,4 +6,6 @@ STRINGTABLE
 BEGIN
     IDS_NTOBJFLD_NAME "NT 物件名稱空間"
     IDS_REGISTRY_NAME "系統登錄"
+    IDS_REGISTRY_COMMENT "Allows browsing of the System Registry"
+    IDS_NTOBJFLD_COMMENT "Allows browsing of the NT Object Namespace"
```
Comment (HBelusca) [inline dll/shellext/ntobjshex/lang/zh-TW.rc:10]:
```suggestion
    IDS_NTOBJFLD_COMMENT "Allows browsing of the NT Object Namespace"
    IDS_REGISTRY_COMMENT "Allows browsing of the System Registry"
```

**Review by HBelusca — APPROVED** [review]
Minor reordering

**Pull request merged**